solidity/contracts/isms/native/OptimismIsm.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
```

Comment:
Did you look at all into the OZ crosschain contracts? There's not a ton going on there but it might be worth inheriting from nonetheless.

Same for Arbitrum, and maybe Polygon as well

Reply:
Interesting. It's the same for what I have for optimism and arbitrum so I can inherit for both.